internal/action/tab.go

```diff
@@ -189,6 +189,55 @@ func (t *TabList) HandleEvent(event tcell.Event) {
 
 				if i := t.LocFromVisual(buffer.Loc{mx, my}); i == -1 {
 					t.List[t.Active()].CurPane().AddTab()
+				} else {
+					anyModified := false
+					for _, p := range t.List[i].Panes {
+						if bp, ok := p.(*BufPane); ok {
+							if bp.Buf.Modified() {
```

Comment:
This produces false negatives if the same buffer is also opened in another pane in another tab (so closing this tab would not cause losing unsaved changes in this buffer).

The similar problem was addressed in #3559 + #3719 by adding `Shared()` which checks if the buffer is opened in more than one pane. Unfortunately we cannot just reuse `Shared()` here, since we need to distinguish the case when all the panes with this buffer are in the tab we are closing (then we need to prompt the user) and the case when at least one such pane is in a different tab (then we don't need to prompt the user).